Update/onik_002.txt

```diff
@@ -3521,7 +3527,7 @@ void main()
 //　ねぐらに向かう鳥たちが、夜の帳がすぐそこまで訪れていることを告げていた￥
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "　ねぐらに向かう鳥たちが、夜の帳がすぐそこまで訪れていることを告げていた。",
-		   NULL, "Birds heading home to roost proclaimed that nightfall would soon be upon us.", Line_Normal);
+		   NULL, "Birds, heading home to roost, proclaimed that nightfall would soon be upon us.", Line_Normal);
```

Comment:
Leave as is; this is either formally or stylistically correct.  Adding commas actually makes subtle changes to the meaning of this sentence.